domainlab/utils/perf_metrics.py

```diff
@@ -44,13 +51,14 @@ def cal_metrics(self, model, loader_te, device, max_batches=None):
                 x_s, y_s = x_s.to(device), y_s.to(device)
                 pred_label, prob, _, *_ = model_local.infer_y_vpicn(x_s)
```

Comment:
@smilesun Could you tell me the returned shape of pred_label? 

Because when inputs are labels, inputs to the classification metrics need to be both in the shape of (N,).

Reply:
Multi-class with logits or probabilities | (N, C) | float | (N,) | int
-- | -- | -- | -- | --

Reply:
i will check it and let you know

Reply:
pred_label.shape
torch.Size([100, 10])

Reply:
that means if we change everything to probability, it should be fine right?

Reply:
Yes. But if we are using probability, do we need to pass a threshold? By default, the  classification metrics function uses threshold=0.5. I don't know if the model's threshold is also the same.

Reply:
I don't think we need to pass threshold, since only the largetst logits will be taken as the predicted label. 
threshold only make since for binary classification AUC. However, I am not sure exaclty how multi-class AUC works